apps/web/playwright/integrations-stripe.e2e.ts

```diff
@@ -69,9 +69,47 @@ test.describe("Stripe integration", () => {
     await expect(page.getByText("Unconfirmed")).toBeVisible();
     await expect(page.getByText("Pending payment").last()).toBeVisible();
   });
+
+  test("Paid booking should be able to be rescheduled", async ({ page, users }) => {
+    const user = await users.create();
+    const eventType = user.eventTypes.find((e) => e.slug === "paid") as Prisma.EventType;
+    await user.apiLogin();
+    await page.goto("/apps/installed");
+
+    await user.getPaymentCredential();
+    await user.setupEventWithPrice(eventType);
+    await user.bookAndPaidEvent(eventType);
+
+    await Promise.all([page.waitForURL("/booking/*"), page.click('[data-testid="reschedule-link"]')]);
+
+    // wait for reschedule page to load
+    await page.waitForSelector('[data-testid="incrementMonth"]');
```

Comment:
this operation it is really needed ? the `page.click` have to wait the selector.
https://playwright.dev/docs/api/class-page#page-click

Reply:
fixed this! pls look at new commits